torch/csrc/jit/grad_executor.h

```diff
@@ -10,7 +10,7 @@ namespace detail {
 
 struct DifferentiableGraphBackwardImpl;
 
-struct DifferentiableGraphBackward : public autograd::Node {
+struct TORCH_API DifferentiableGraphBackward : public autograd::Node {
```

Comment:
I have to expose this as `TORCH_API` otherwise the there will be `undefined symbol: _ZTIN5torch3jit6detail27DifferentiableGraphBackwardE` at loading time.